src/core/particle_data.hpp

```diff
@@ -62,20 +62,10 @@ enum {
 /** \ref ParticleProperties::ext_flag "ext_flag" value for particle subject to
  *  an external force
  */
-#define PARTICLE_EXT_FORCE 1
-/** \ref ParticleProperties::ext_flag "ext_flag" value for fixed coordinate
- *  coord. */
```

Comment:
did you remove the correct docstring?

Reply:
No, I'll address that in a separate pull request.